constants/styles/MainScreen.ts

```diff
@@ -44,7 +45,7 @@ export const styles = (isDark: boolean) => StyleSheet.create({
     },
     dataPerishableGoodsTextInput: {
         textAlign: "center",
-        paddingBottom: (Platform.OS === 'android') ? PixelRatio.getPixelSizeForLayoutSize(6.5) : 0,
+        // paddingBottom: (Platform.OS === 'android') ? PixelRatio.getPixelSizeForLayoutSize(6.5) : 0,
```

Comment:
It decreases the height of "PERISHABLE GOODS" control and is no longer equal to the height of "EXPIRATION DATE". What did you want to solve with this?

Reply:
> It decreases the height of "PERISHABLE GOODS" control and is no longer equal to the height of "EXPIRATION DATE". What did you want to solve with this?

It was taking up extra space in the input, which was not a problem in general but the space needed to be reduced for small sized android devices.

Reply:
> > It decreases the height of "PERISHABLE GOODS" control and is no longer equal to the height of "EXPIRATION DATE". What did you want to solve with this?
> 
> It was taking up extra space in the input, which was not a problem in general but the space needed to be reduced for small sized android devices.

Could you attach a screenshot, please? Thank you so much in advance

Reply:
> > > It decreases the height of "PERISHABLE GOODS" control and is no longer equal to the height of "EXPIRATION DATE". What did you want to solve with this?
> > 
> > 
> > It was taking up extra space in the input, which was not a problem in general but the space needed to be reduced for small sized android devices.
> 
> Could you attach a screenshot, please? Thank you so much in advance

![ss](https://user-images.githubusercontent.com/55059285/90006522-ae9e7400-dcb6-11ea-9c9b-cff65ca09137.jpeg)
Here it is. My aim was to basically reduce the spacing above the button, it looked good even after removing the padding so I decided to try that out.

Reply:
It is okay, but both inputs should have the same height, moreover this must be equal to the height of each inputs from the user activity. If we need more space then I would try [this](https://github.com/ivgraai/expiry-util/pull/107#discussion_r469150588).
(Over and above please do not forget to vertically center the placeholder in the box!)

Reply:
Okay!

Reply:
Any update on vertically centering?